docs/howto/operator/google/transfer/sheets_to_gcs.rst

```diff
@@ -32,7 +32,7 @@ common spreadsheet tasks.
 Prerequisite Tasks
 ^^^^^^^^^^^^^^^^^^
 
-.. include:: /howto/operator/google/_partials/prerequisite_tasks.rst
+.. include::/howto/operator/google/_partials/prerequisite_tasks.rst
```

Comment:
This might not work with Sphinx

Reply:
All the directives in Sphinx have a space between them so not sure

Reply:
@kaxil the docs are building, do you have more concerns?

Reply:
Unfortunately, the docs-upload step had failed (https://github.com/apache/airflow/pull/10962/checks?check_run_id=1119863837#step:5:25), so can't see the rendered docs (rebasing to latest master should help) but if they are rendering fine than I have no issues at all.

Reply:
I think this turns it in to a comment.